pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -1009,29 +1217,42 @@ def continuous_sweep(self):
     )
 
     def fft(self, source, destination, window):
-        """
-        The FFT command performs a discrete Fourier transform on the source trace array and stores
-        the logarithms of the magnitudes of the results in the destination array. The maximum length
-        of any of the traces is 601 points.
-        Takes type 'Trace' or 'TRA', 'TRB'
-        FFT is designed to be used in transforming zero-span amplitude-modulation information into
-        the frequency domain. Performing an FFT on a frequency sweep will not provide time-domain
-        results. The FFT results are displayed on the spectrum analyzer in a logarithmic
-        amplitude scale. For the horizontal dimension, the frequency at the left side of the
-        graph is 0 Hz, and at the right side is Finax- Fmax is equal to 300 divided by sweep time.
-        As an example, if the sweep time of the analyzer is 60 ms, Fmax equals 5 kHz.
-        The FFT algorithm assumes that the sampled signal is periodic with an integral number of
-        periods within the time-record length (that is, the sweep time of the analyzer). Given this
-        assumption, the transform computed is that of a time waveform of infinite duration, formed
-        of concatenated time records. In actual measurements, the number of periods of the sampled
-        signal within the time record may not be integral. In this case, there is a step
-        discontinuity at the intersections of the concatenated time records in the assumed time
-        waveform of infinite duration. This step discontinuity causes measurement errors,
-        both amplitude uncertainty (where the signal level appears to vary with small changes in
-        frequency) and frequency resolution (due to filter shape factor and sidelobes). Windows
-        are weighting functions that are applied to the input data to force the ends of that
-        data smoothly to zero, thus reducing the step discontinuity and reducing measurement
-        errors.
+        """The FFT command performs a discrete Fourier transform on the source
+        trace array and stores the logarithms of the magnitudes of the results
+        in the destination array. The maximum length of any of the traces is
+        601 points. FFT is designed to be used in transforming zero-span
+        amplitude-modulation information into the frequency domain. Performing
```

Comment:
A docstring should start with a one-line summary (in imperative mode) and than more context:
```suggestion
        """Calculate and show a discrete Fourier transform.

        The FFT command performs a discrete Fourier transform on the source
        trace array and stores the logarithms of the magnitudes of the results
        in the destination array. The maximum length of any of the traces is
        601 points. FFT is designed to be used in transforming zero-span
        amplitude-modulation information into the frequency domain. Performing
```